composer.json

```diff
@@ -22,7 +22,7 @@
 	"require": {
 		"php": ">=5.3.0",
 		"data-values/data-values": "~1.0|~0.1",
-		"data-values/interfaces": "^0.1.5",
+		"data-values/interfaces": "~0.2.0|^0.1.5",
```

Comment:
This was wrong before the change. Should be `~0.1.5`. Else it's `>=0.1.5 AND <1.0.0`, which happens to include the added range here, which thus has no effect